geometry_msgs/PoseStamped left_tcp
geometry_msgs/PoseStamped right_tcp
---
